### PR TITLE
Feature/create management section

### PR DESCRIPTION
**Pull request by mthanh2209**

### CODE REVIEW

File: practice_two/src/styles/sections/management.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.management {
+	display: grid;
```
Comment (hungnguyenhuu-agilityio) [inline practice_two/src/styles/sections/management.css]: I don't see any properties relate to the grid layout you set? Any wrong here?

Comment (mthanh2209) [reply practice_two/src/styles/sections/management.css]: Sorry for the confusion here, i fixed it

File: practice_two/src/styles/sections/management.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
+.management {
+	margin-top: 3.0625rem;
+}
+
+.management-img {
+	margin: 0 auto;
+	width: calc(349 / 16 * 1rem);
+	height: calc(306 / 16 * 1rem);
```
Comment (hungnguyenhuu-agilityio) [inline practice_two/src/styles/sections/management.css:8]: what are 349, 306, 16? I think you should think about the solution to use calc

Comment (mthanh2209) [reply practice_two/src/styles/sections/management.css:8]: Sorry. I will check again

**Pull request closed without merging**